CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed 🔧
 
 * Changed default font to [Ubuntu-Light](https://fonts.google.com/specimen/Ubuntu).
+* FileStorage::from_path now takes `Into<Path>` instead of `String`
```

Comment:
This changelog is only for the `egui` crate. This is not very well explained, and very confusing since `egui_glium` doesn't even have it's own changelog :) So I just added one in https://github.com/emilk/egui/commit/8c82eb1970a00aa65506aed102141dabbd3a3f8b

You can move the changelog line to `egui_glium/CHANGELOG.md`, but it's also fine to just remove this (since it is a very minor change for a crate that is not that well used ... yet)